src/hooks/usePokemons.ts

```diff
@@ -32,8 +45,30 @@ const usePokemons = () => {
       };
     }) ?? [];
 
+  const handleLoadMore = async () => {
+    if (loadedAll) {
+      return;
+    }
+    setLoadingMore(true);
+    try {
+      const result = await fetchMore({
```

Comment:
what will happen if this call will fail?

Reply:
nothing IMO in this case seems to be ok